src/web/aws-cloudwatch/StepReview.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { Button, Col } from 'react-bootstrap';
+
+export default class Review extends Component {
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired,
+    getAllValues: PropTypes.func.isRequired,
+  }
+
+  render() {
+    const { getAllValues, onSubmit } = this.props;
+
+    return (
```

Comment:
This step has no title, being inconsistent with how others are displayed.

Reply:
Just hadnt gotten around to styling this page, added a header for consistency for now